.github/workflows/ci.yml

```diff
@@ -36,8 +36,28 @@ jobs:
           version: "${{ matrix.NXF_VER }}"
 
       - name: Run pipeline with test data
-        # TODO nf-core: You can customise CI pipeline run tests as required
-        # For example: adding multiple test runs with different parameters
-        # Remember that you can parallelise this by using strategy.matrix
         run: |
-          nextflow run ${GITHUB_WORKSPACE} -profile test,docker --outdir ./results
+          nextflow run ${GITHUB_WORKSPACE} -profile test_targeted,docker --outdir ./results_targeted
+
+  test_screening:
```

Comment:
You could simplify calling these commands several times by adding a matrix: 

```
jobs:
  test:
    name: Run pipeline with test data
    ...
    strategy:
      matrix:
        NXF_VER:
          - "22.10.1"
          - "latest-everything"
        config:
          - "test"
          - "test_screening"
          - "test_targeted"
    steps:
      - name: Check out pipeline code
        uses: actions/checkout@v3

      - name: Install Nextflow
        uses: nf-core/setup-nextflow@v1
        with:
          version: "${{ matrix.NXF_VER }}"

      - name: Run pipeline with test data
        run: |
          nextflow run ${GITHUB_WORKSPACE} -profile ${{ matrix.config }},docker --outdir ./results
```

Reply:
Thanks for the suggestion! 🥳